Description: disable tests which require a network connection
Author: Jeremy Lainé <jeremy.laine@m4x.org>

--- a/tests/tests.pro
+++ b/tests/tests.pro
@@ -2,12 +2,10 @@ TEMPLATE = subdirs
 SUBDIRS = \
     qxmpparchiveiq \
     qxmppbindiq \
-    qxmppcallmanager \
     qxmppcarbonmanager \
     qxmppdataform \
     qxmppdiscoveryiq \
     qxmppentitytimeiq \
-    qxmppiceconnection \
     qxmppiq \
     qxmppjingleiq \
     qxmppmammanager \
@@ -27,7 +25,6 @@ SUBDIRS = \
     qxmppstanza \
     qxmppstreamfeatures \
     qxmppstunmessage \
-    qxmpptransfermanager \
     qxmpputils \
     qxmppvcardiq \
     qxmppversioniq
